internal/servers/controller/cors_test.go

```diff
@@ -38,6 +38,10 @@ kms "aead" {
 		key_id = "global_worker-auth"
 }
 
+listener "tcp" {
+		purpose = "cluster"
```

Comment:
This indentation seems off compared to the other blocks here.

Reply:
Could you share what you're seeing on your end just to make sure? The only difference my editor reports is that I actually used tabs there and not spaces.

Reply:
Ok that is probably why. It looks like there is a mix of spaces and tabs within the raw string. We should probably just use one or the other here, but not an issue that needs to be solved now.